userProfile/userProfile.mjs

```diff
@@ -0,0 +1,68 @@
+
+import DatabaseDriver from "../database/driver.mjs"
+
+// This module will use the TinyPEN API (RERUM Mongo DB)
+const database = new DatabaseDriver("mongo")
+
+export async function findUserById(id = null) {
+  try {
+    
+        let userProfile  = await database.find({ id: id })
```

Comment:
No need to do this `database.find()` if you are minting a mock user below.  The return to this is either a user object or an error which gets passed forward.  You have to choose whether to use this and pass it forward correctly, or not to use it in favor of the mock object.  For the scope of this issue, I recommend avoiding `database.find()` as that will bloat the scope of this issue because you will need to make sure it works.

Reply:
Updated and tested. Thanks for the suggestion; it's really appreciated!